teachertool/src/state/appStateContext.tsx

```diff
@@ -14,8 +14,8 @@ const copilotSlot = url.match(/copilot=([^&]+)/);
 const copilotEndpoint =
     copilotSlot && copilotSlot[1]
         ? copilotSlot[1] === "local"
-            ? `${DEV_BACKEND_LOCALHOST}/api`
-            : `https://makecode-ppe-app-backend-eastus-${copilotSlot[1]}.azurewebsites.net/api`
```

Comment:
Hmm shouldn't this point to prod\staging depending on the URL?

Reply:
By default, it points to the same api root used by the teacher tool for all its requests (https://github.com/microsoft/pxt/blob/master/teachertool/src/index.tsx#L53). This only takes effect if the ?copilot=<alias> query parameter is present in the URL. It's just what allowed us to use the ?copilot=thsparks flag for all those demos and testing scenarios, where we wanted to go through our backend but hadn't shipped the backend apis yet. It's useful for testing, but we will probably want to get rid of it at some point.

Reply:
Sounds good.